src/doc/guide.md

```diff
@@ -556,11 +552,7 @@ Rust's take on `if` is not particularly complex, but it's much more like the
 `if` you'll find in a dynamically typed language than in a more traditional
 systems language. So let's talk about it, to make sure you grasp the nuances.
 
-`if` is a specific form of a more general concept, the 'branch.' The name comes
-from a branch in a tree: a decision point, where depending on a choice,
-multiple paths can be taken.
```

Comment:
My reasoning: This is primarily trivia, not about Rust, and programmers should already know it, so it should be removed.